recipe/meta.yaml

```diff
@@ -22,13 +22,13 @@ requirements:
         - python  # [win]
         - cmake  # [win]
         - msinttypes  # [win and py<35]
-        - jpeg
-        - zlib
         - bison  # [unix]
         - flex  # [unix]
+        - zlib
+        - jpeg 9b
```

Comment:
👍 - better than just relying on the latest version, but still not the panacea that is conda-build pinning 😄